tools/run-s3gw.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
+#
+# run-s3gw.sh - wrapper to run a radosgw binary
+# Copyright 2022 SUSE, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+default_id="s3gw"
+default_debug_level="none"
+
+rgw_pid=
+
+usage() {
+  cat <<EOF
+usage: $0 [options... -- args]
+
+options:
+  --help                Shows this message.
+  --id ID               Specifies a custom instance ID (default: ${default_id}).
+  --cert FILE KEY       Specifies SSL certificate. Expects a .crt as first
+                        argument, and a .key as second argument.
+  --dns-name VALUE      For vhost style buckets, VALUE should be the DNS domain
+                        to be used.
+  --debug LEVEL         Runs with debug. Levels available: "high", "medium",
+                        "low", "none" (default: ${default_debug_level}).
+  --no-telemetry        Disable telemetry.
+  --telemetry-url URL   Specifies telemetry URL.
+  --with-status         Enables status frontend at port 9090.
```

Comment:
The docker file says `EXPOSE 7481`, not `9090`

Reply:
7481, I think, is the SSL port. I didn't add the 9090 to the dockerfile, wonder if I should even if that's merely for documentation purposes.